tests/test_cli.py

```diff
@@ -197,9 +197,14 @@ def test_log_encoding(self, caplog, monkeypatch):
                 self.exec_cli(command)
 
         # first message should be 'log encoding: <encoding>'
-        match = caplog.records()[0].message.split(':')
+        match = None
+        for record in caplog.records():
```

Comment:
Wasn't sure about that one myself, haven't hit a problem with it personally. Okay, have dropped that commit from the branch, leaving just the bit to force log format (I've seen things from osbs logged w/o the proper format) and the flush.